src/bo4egenerator/cli.py

```diff
@@ -0,0 +1,55 @@
+"""
+contains CLI logic for bo4e-dotnet-generator.
+this is the main entry point for the bo4e-generator. It generates C# classes from the BO4E schema files.
+"""
+
+import os
+import platform
+from pathlib import Path
+
+import typer
+
+from bo4egenerator.generator import generate_csharp_classes
+from bo4egenerator.tooling import install_bo4e_schema_tool
+
+app = typer.Typer(help="It generates C# classes from the BO4E schema files.")
+
+
+@app.command()
+def main() -> None:
+    """
+    It will install the BO4E-Schema-Tool and
+    generate C# classes from the BO4E schema files with help of Quicktype.
+    """
+    # Define the base directories
+    project_root = os.path.abspath(os.path.dirname(__file__))  # Root directory of the project
```

Comment:
ich finde pathlib angenehmer.
```suggestion
    project_root = Path.cwd()
```
mit `from pathlib impor Path`